include/psa/crypto.h

```diff
@@ -871,7 +871,7 @@ psa_status_t psa_hash_compute(psa_algorithm_t alg,
  *                          such that #PSA_ALG_IS_HASH(\p alg) is true).
  * \param[in] input         Buffer containing the message to hash.
  * \param input_length      Size of the \p input buffer in bytes.
- * \param[out] hash         Buffer containing the expected hash value.
+ * \param[in] hash          Buffer containing the expected hash value.
```

Comment:
Thanks! Can you please do `psa_mac_verfiy` while you're at it?

Reply:
Should we also fix `signature` in `psa_verify_message()`? (that is also `const` but documented as `out`)